src/web/page/templates.rs

```diff
@@ -84,7 +89,16 @@ fn load_rustc_resource_suffix(conn: &Connection) -> Result<String> {
 }
 
 pub(super) fn load_templates(conn: &Connection) -> Result<Tera> {
-    let mut tera = Tera::new("tera-templates/**/*")?;
+    // This uses a custom function to find the templates in the filesystem instead of Tera's
+    // builtin way (passing a glob expression to Tera::new), speeding up the startup of the
+    // application and running the tests.
+    //
+    // The problem with Tera's template loading code is, it walks all the files in the current
+    // directory and matches them against the provided glob expression. Unfortunately this means
+    // Tera will walk all the rustwide workspaces, the git repository and a bunch of other
+    // unrelated data, slowing down the search a lot.
```

Comment:
I'm going to try and get this upstreamed to globwalk. This workaround is fine in the meantime though.
```suggestion
    // The problem with Tera's template loading code is, it walks all the files in the current
    // directory and matches them against the provided glob expression. Unfortunately this means
    // Tera will walk all the rustwide workspaces, the git repository and a bunch of other
    // unrelated data, slowing down the search a lot.
    //
    // TODO: remove this when https://github.com/Gilnaa/globwalk/issues/29 is fixed
```